src/test/java/xyz/vegaone/easytracking/Sprint/SprintServiceTest.java

```diff
@@ -0,0 +1,121 @@
+
+        //when
+        Sprint savedSprint = buildAndSaveSprint();
+
+        //then
+        Assert.assertNotNull("There should have been one Sprint saved in the database.", savedSprint);
+        Assert.assertEquals("The sprint end date should have matched", SPRINT_END_DATE, savedSprint.getEndDate());
+        Assert.assertEquals("The sprint end date should have matched", SPRINT_START_DATE, savedSprint.getStartDate());
+        Assert.assertEquals("The sprint end date should have matched", SPRINT_NUMBER, savedSprint.getSprintNumber());
+    }
+
+    @Test
+    public void getSprintTest() {
+        //given
+        Sprint savedSprint = sprintService.createSprint(buildAndSaveSprint());
+
+        UserStory userStory = new UserStory();
+        userStory.setTitle(TITLE);
+        userStory.setDescription(DESCRIPTION);
+        userStory.setOwner(OWNER);
+        userStory.setEstimation(ESTIMATION);
+        userStory.setPriority(PRIORITY);
+        userStory.setStatus(STATUS);
+        userStory.setProjectId(PROJECT_ID);
+        userStory.setSprintId(SPRINT_ID);
+
+        UserStory savedUserStory = userStoryService.createUserStory(userStory);
+
+        //when
+        Sprint findSprint = sprintService.getSprint(savedSprint.getId());
+
+        //then
+        Assert.assertNotNull("There should have been one Sprint saved in the database.", findSprint);
+        Assert.assertEquals("The sprint end date should have matched", SPRINT_END_DATE, findSprint.getEndDate());
+        Assert.assertEquals("The sprint end date should have matched", SPRINT_START_DATE, findSprint.getStartDate());
+        Assert.assertEquals("The sprint end date should have matched", SPRINT_NUMBER, findSprint.getSprintNumber());
+        Assert.assertEquals("There should have been a userstory associated with the sprint", 1, findSprint.getUserStoryList().size());
+
+    }
+
+    @Test
+    public void deleteSpring() {
+        //given
+        Sprint savedSprint = buildAndSaveSprint();
+
+        //when
+        sprintService.deleteSprint(savedSprint.getId());
+
+        //then
+        Assert.assertNull("The sprint should have been deleted from the db", sprintService.getSprint(savedSprint.getId()));
+    }
+
+    @Test
+    public void updateSprintTest() {
+        //given
+        Sprint savedSprint = buildAndSaveSprint();
+
+        //when
+        savedSprint.setSprintNumber(NEW_SPRINT_NUMBER);
+        Sprint updatedSprint = sprintService.updateSprint(savedSprint);
+
+        //then
+        Assert.assertNotNull("There should have been one sprint updated in the database.", updatedSprint);
+        Assert.assertEquals("The userStory title should have matched", NEW_SPRINT_NUMBER, updatedSprint.getSprintNumber());
+    }
+
+    private Sprint buildAndSaveSprint() {
+        Sprint sprint = new Sprint();
+        sprint.setEndDate(SPRINT_END_DATE);
+        sprint.setStartDate(SPRINT_START_DATE);
+        sprint.setSprintNumber(SPRINT_NUMBER);
+
+        //when
```

Comment:
We use //given //when //then  only in test methods, not in private methods. This comment should be removed.

Reply:
@sabaubogdan ^